MediaSystem.cpp

```diff
@@ -209,10 +229,14 @@ class PlayReady : public IMediaKeys, public IMediaKeysExt {
         SafeCriticalSection lock(drmAppContextMutex_);
 
         // if secure stop is not supported, return NotAllowed
-        DRM_BOOL supported = Drm_SupportSecureStop();
+        DRM_BOOL supported = FALSE;
+#ifdef NETFLIX
+        supported = Drm_SupportSecureStop();
+#endif
         if (supported == FALSE)
             return CDMi_SUCCESS;
```

Comment:
Playready Secure Stop feature should be independent of Netflix support. You may want to use the Playready plugin with secure stop feature for other OTT video services for example. If this isn't supported doesn't it just return anyway?